### PR TITLE
#1055 Added Held fee paid judicial role into the export file

### PR DESCRIPTION
**Pull request by mbrookeswebdev**

### CODE REVIEW

File: tests/unit/filters.spec.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  it('returns string `Prefer not to say`', () => {
+    expect(heldFeePaidJudicialRole('prefer-not-to-say')).toBe('Prefer not to say');
+  });
+});
+
+describe('when input value is boolean `false`', () => {
+  it('returns string `No`', () => {
+    expect(heldFeePaidJudicialRole(false)).toBe('No');
+  });
+});
```
Comment (warrensearle) [inline tests/unit/filters.spec.js:31]: What is expected output when input value is empty/null?

**Review by warrensearle — APPROVED** [review]
This looks good. 👍  
See comment/question with regards empty/null value.

File: tests/unit/filters.spec.js (hunk 1 above)
Comment (mbrookeswebdev) [reply tests/unit/filters.spec.js:31]: Yes, I was not sure how to deal with that. Technically we could only receive five pre-defined values - (three types of judicial role (as a Yes), No and prefer-not-to-say). At the moment empty/null is covered by 'Prefer not to say', which is essentially 'Unknown'. Should I cover this case in the function and return 'Unknown' if somehow value is null/empty?

Comment (warrensearle) [reply tests/unit/filters.spec.js:31]: I'm thinking the filter should probably return an empty string when input is null/empty and only return 'Prefer not to say' when the candidate has chosen that option. Please double check with Becky if you like

Comment (mbrookeswebdev) [reply tests/unit/filters.spec.js:31]: @warrensearle I've checked with Becky, she said to return 'Unknown', so I've made the changes.

**Review by warrensearle — APPROVED** [review]
Excellent, thanks @mbrookeswebdev 👍

**Pull request merged**